tcms_settings_dir/multi_tenant.py

```diff
@@ -53,3 +53,6 @@
 
 # everybody can access the main instance
 SHARED_APPS = INSTALLED_APPS  # noqa: F821
+
+# add tennants context processor
+TEMPLATES[OPTIONS[context_processors]].append("tenant_navbar_processor")  # noqa: F821
```

Comment:
Indexing !!!

Note: settings files are actually loaded multiple times so you need to ensure the context processor is configured only once. See the if conditions above in this file.